charts/back8sup/Chart.yaml

```
@@ -0,0 +1,23 @@
+apiVersion: v2
+name: back8sup
+description: A Helm chart for Kubernetes
```

```suggestion
description: A Helm chart for Kubernetes
home: https://github.com/adfinis-sygroup/back8sup
```

Chances are that fixing this will make the charts-testing action notice more stuff.